### DISCUSSION TITLE
Why can't I create a appwrite collection using flutter?

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [appwrite/appwrite#5193] [Q&A]
### 👟 Reproduction steps

appwrite does not have the functionality in flutter framework from whch we can create collection and delete collection .

If anyone has any lead on this please let me know.

### 👍 Expected behavior

🐛 Bug Report: I can not create a appwrite collection using flutter .

### 👎 Actual Behavior

🐛 Bug Report: I can not create a appwrite collection using flutter .

### 🎲 Appwrite version

Version 1.2.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@hemangjoshi37a, thanks for opening this issue! 🙏This is actually the expected behavior. Giving a client SDK access to create collections is like giving it access to create any database table they want which can create clutter and open up channels for abuse. If you really want to create collections at runtime, I suggest using a server SDK in an Appwrite Function. The Appwrite Function would allow you to create the collection you want to create rather than letting the client make any collection they want.

**Reply by hemangjoshi37a** [reply]
Yes but at least we should have that option in flutter to make, update and delete collection.

**Reply by stnguyen90** [reply]
Again, allowing update and delete client side is a security problem and Appwrite is secure by default.

**Reply by hemangjoshi37a** [reply]
There is nothing really insecure about this. It is just an inconvenience
